2023-03-17 08:26:53,016 INFO    MainThread:4818 [wandb_setup.py:_flush():68] Configure stats pid to 4818
2023-03-17 08:26:53,016 INFO    MainThread:4818 [wandb_setup.py:_flush():68] Loading settings from /home/kidimerek/.config/wandb/settings
2023-03-17 08:26:53,017 INFO    MainThread:4818 [wandb_setup.py:_flush():68] Loading settings from /home/kidimerek/Documents/Studium/Thesis/agnostic_rl-main/agent/wandb/settings
2023-03-17 08:26:53,017 INFO    MainThread:4818 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-03-17 08:26:53,017 INFO    MainThread:4818 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-03-17 08:26:53,017 INFO    MainThread:4818 [wandb_init.py:_log_setup():492] Logging user logs to /home/kidimerek/Documents/Studium/Thesis/agnostic_rl-main/agent/wandb/run-20230317_082653-z92qik1q/logs/debug.log
2023-03-17 08:26:53,017 INFO    MainThread:4818 [wandb_init.py:_log_setup():493] Logging internal logs to /home/kidimerek/Documents/Studium/Thesis/agnostic_rl-main/agent/wandb/run-20230317_082653-z92qik1q/logs/debug-internal.log
2023-03-17 08:26:53,018 INFO    MainThread:4818 [wandb_init.py:init():532] calling init triggers
2023-03-17 08:26:53,018 INFO    MainThread:4818 [wandb_init.py:init():538] wandb.init called with sweep_config: {}
config: {}
2023-03-17 08:26:53,018 INFO    MainThread:4818 [wandb_init.py:init():554] re-initializing run, found existing run on stack: uef9a4po
2023-03-17 08:26:53,020 INFO    MainThread:4818 [wandb_run.py:_finish():1854] finishing run agnostic/rest_run_pong/uef9a4po
2023-03-17 08:26:53,020 ERROR   MainThread:4818 [jupyter.py:save_history():441] Run pip install nbformat to save notebook history
2023-03-17 08:26:53,020 INFO    MainThread:4818 [jupyter.py:save_ipynb():379] not saving jupyter notebook
2023-03-17 08:26:53,020 INFO    MainThread:4818 [wandb_init.py:_jupyter_teardown():420] cleaning up jupyter logic
2023-03-17 08:26:53,020 INFO    MainThread:4818 [wandb_run.py:_atexit_cleanup():2083] got exitcode: 0
2023-03-17 08:26:53,020 INFO    MainThread:4818 [wandb_run.py:_restore():2066] restore
2023-03-17 08:26:53,022 INFO    MainThread:4818 [wandb_run.py:_restore():2072] restore done
2023-03-17 08:26:58,972 INFO    MainThread:4818 [wandb_run.py:_footer_history_summary_info():3442] rendering history
2023-03-17 08:26:58,972 INFO    MainThread:4818 [wandb_run.py:_footer_history_summary_info():3474] rendering summary
2023-03-17 08:26:58,985 INFO    MainThread:4818 [wandb_run.py:_footer_sync_info():3398] logging synced files
2023-03-17 08:26:59,006 INFO    MainThread:4818 [wandb_init.py:init():588] starting backend
2023-03-17 08:26:59,006 INFO    MainThread:4818 [wandb_init.py:init():592] setting up manager
2023-03-17 08:26:59,012 INFO    MainThread:4818 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-03-17 08:26:59,016 INFO    MainThread:4818 [wandb_init.py:init():599] backend started and connected
2023-03-17 08:26:59,021 INFO    MainThread:4818 [wandb_run.py:_label_probe_notebook():1203] probe notebook
2023-03-17 08:26:59,021 INFO    MainThread:4818 [wandb_run.py:_label_probe_notebook():1213] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-03-17 08:26:59,021 INFO    MainThread:4818 [wandb_init.py:init():687] updated telemetry
2023-03-17 08:26:59,065 INFO    MainThread:4818 [wandb_init.py:init():727] communicating run to backend with 60.0 second timeout
2023-03-17 08:27:00,414 INFO    MainThread:4818 [wandb_run.py:_on_init():2134] communicating current version
2023-03-17 08:27:00,479 INFO    MainThread:4818 [wandb_run.py:_on_init():2143] got version response upgrade_message: "wandb version 0.14.0 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-03-17 08:27:00,479 INFO    MainThread:4818 [wandb_init.py:init():775] starting run threads in backend
2023-03-17 08:27:00,552 INFO    MainThread:4818 [wandb_run.py:_console_start():2114] atexit reg
2023-03-17 08:27:00,552 INFO    MainThread:4818 [wandb_run.py:_redirect():1969] redirect: SettingsConsole.WRAP_RAW
2023-03-17 08:27:00,552 INFO    MainThread:4818 [wandb_run.py:_redirect():2034] Wrapping output streams.
2023-03-17 08:27:00,552 INFO    MainThread:4818 [wandb_run.py:_redirect():2059] Redirects installed.
2023-03-17 08:27:00,553 INFO    MainThread:4818 [wandb_init.py:init():817] run started, returning control to user process
2023-03-17 08:29:27,085 INFO    MainThread:4818 [wandb_setup.py:_flush():68] Configure stats pid to 4818
2023-03-17 08:29:27,085 INFO    MainThread:4818 [wandb_setup.py:_flush():68] Loading settings from /home/kidimerek/.config/wandb/settings
2023-03-17 08:29:27,086 INFO    MainThread:4818 [wandb_setup.py:_flush():68] Loading settings from /home/kidimerek/Documents/Studium/Thesis/agnostic_rl-main/agent/wandb/settings
2023-03-17 08:29:27,086 INFO    MainThread:4818 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-03-17 08:29:27,086 INFO    MainThread:4818 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-03-17 08:29:27,086 INFO    MainThread:4818 [wandb_init.py:_log_setup():492] Logging user logs to /home/kidimerek/Documents/Studium/Thesis/agnostic_rl-main/agent/wandb/run-20230317_082927-jymwx5lz/logs/debug.log
2023-03-17 08:29:27,086 INFO    MainThread:4818 [wandb_init.py:_log_setup():493] Logging internal logs to /home/kidimerek/Documents/Studium/Thesis/agnostic_rl-main/agent/wandb/run-20230317_082927-jymwx5lz/logs/debug-internal.log
2023-03-17 08:29:27,087 INFO    MainThread:4818 [wandb_init.py:_jupyter_setup():438] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f165cccbdf0>
2023-03-17 08:29:27,087 INFO    MainThread:4818 [wandb_init.py:init():532] calling init triggers
2023-03-17 08:29:27,087 INFO    MainThread:4818 [wandb_init.py:init():538] wandb.init called with sweep_config: {}
config: {}
2023-03-17 08:29:27,087 INFO    MainThread:4818 [wandb_init.py:init():554] re-initializing run, found existing run on stack: z92qik1q
2023-03-17 08:29:27,091 INFO    MainThread:4818 [wandb_run.py:_finish():1854] finishing run agnostic/rest_run_pong/z92qik1q
2023-03-17 08:29:27,091 INFO    MainThread:4818 [wandb_run.py:_atexit_cleanup():2083] got exitcode: 0
2023-03-17 08:29:27,091 INFO    MainThread:4818 [wandb_run.py:_restore():2066] restore
2023-03-17 08:29:27,092 INFO    MainThread:4818 [wandb_run.py:_restore():2072] restore done
2023-03-17 08:29:32,080 INFO    MainThread:4818 [wandb_run.py:_footer_history_summary_info():3442] rendering history
2023-03-17 08:29:32,080 INFO    MainThread:4818 [wandb_run.py:_footer_history_summary_info():3474] rendering summary
2023-03-17 08:29:32,093 INFO    MainThread:4818 [wandb_run.py:_footer_sync_info():3398] logging synced files
